app/printing/printFull/get.controller.test.js

```diff
@@ -1,36 +1,213 @@
 const { printFullSentencePlan } = require('./get.controller')
+const { getSentencePlan } = require('../../../common/data/sentencePlanningApi')
+
+jest.mock('../../../common/data/sentencePlanningApi')
 
 const tokens = { authorisationToken: 'mytoken' }
+const sentencePlanEmpty = {}
+const sentencePlan = require('../../../mockServer/sentencePlans/6.json')
 
 describe('printFullSentencePlan', () => {
   const req = {
     path: 'lead/me/up/the/garden/path',
     params: {
       id: 1,
+      planId: 6,
     },
     tokens,
   }
   const res = {
     render: jest.fn(),
   }
 
+  beforeEach(() => {
+    getSentencePlan.mockReset()
+  })
+
   describe('show print page', () => {
-    it('should set the correct render values n', async () => {
+    it('should set the correct render values when getting a full sentence plan', async () => {
+      let expectedObjectives = {
```

Comment:
This is quite a complex expect object declaration. I think there is a case to be made for isolating some of the code blocks and testing those portions and then maybe requiring an external _ expectedObjectives_ object.